test/functionalTest/cases/1170_get_types_values_option/get_types_values_option_with_null_and_empty_types.test

```diff
@@ -58,15 +58,15 @@ mongoCmd ${CB_DB_NAME} 'db.entities.insert({
                 "type": "",
                 "servicePath": "/"
         },
-        "attrNames": ["A1", "A3"],
```

Comment:
This ha no influence in the test outcome, but fix a typo (see echo line for this step)

NTC (informative)